src/libbacktrace/Makefile.am

```diff
@@ -6,12 +6,12 @@
 # met:
 
 #     (1) Redistributions of source code must retain the above copyright
-#     notice, this list of conditions and the following disclaimer. 
```

Comment:
Be careful with markdown files! I can't give this a proper review right now, but trailing white space is significant in them, it's used in the book intentionally at least once.


On Mar 13, 2016, 16:16 -0400, rkruppe<notifications@github.com>, wrote:
>  
> Insrc/libbacktrace/Makefile.am(https://github.com/rust-lang/rust/pull/32235#discussion_r55942955):
>  
> >@@ -6,12 +6,12 @@># met:>>#   (1) Redistributions of source code must retain the above copyright>-#   notice, this list of conditions and the following disclaimer.
>  
> libbracktrace is not part of Rust proper, so the Rust styleguide doesn't apply (and these files shouldn't be modified anyway if at all possible, because it complicates upgrades).
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/rust-lang/rust/pull/32235/files#r55942955).
>  
>  
>